include/grpc++/credentials.h

```diff
@@ -44,9 +44,17 @@ class ChannelArguments;
 class Channel;
 class SecureCredentials;
 
+/// A credentials object encapsulates all the state needed by a client to
+/// authenticate with a server and make various assertions, e.g., about the
+/// client’s identity, role, or whether it is authorized to make a particular
+/// call.
+///
+/// \see https://github.com/grpc/grpc-common/blob/master/grpc-auth-support.md
```

Comment:
Oh. I missed the repo. Aren't we deprecating grpc-common anyway?

On Wed, Aug 26, 2015 at 9:54 AM David G. Quintas <notifications@github.com>
wrote:

> In include/grpc++/credentials.h
> <https://github.com/grpc/grpc/pull/3074#discussion_r38005820>:
>
> > @@ -44,9 +44,17 @@ class ChannelArguments;
> >  class Channel;
> >  class SecureCredentials;
> >
> > +/// A credentials object encapsulates all the state needed by a client to
> > +/// authenticate with a server and make various assertions, e.g., about the
> > +/// client’s identity, role, or whether it is authorized to make a particular
> > +/// call.
> > +///
> > +/// \see https://github.com/grpc/grpc-common/blob/master/grpc-auth-support.md
>
> that'd be great. Is it able to do that even if the file lives in a
> different repo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/3074/files#r38005820>.
>